go.mod

```diff
@@ -7,14 +7,15 @@ require (
 	cloud.google.com/go/pubsub v1.3.1
 	cloud.google.com/go/storage v1.6.0
 	github.com/crossplane/crossplane v0.13.0-rc.0.20200924053741-18324aa15a19
-	github.com/crossplane/crossplane-runtime v0.9.1-0.20201005140116-dc04a11d0062
-	github.com/crossplane/crossplane-tools v0.0.0-20201001224552-fb258cc0eb30
+	github.com/crossplane/crossplane-runtime v0.9.1-0.20201008065523-51c117eff562
+	github.com/crossplane/crossplane-tools v0.0.0-20201007233256-88b291e145bb
 	github.com/google/go-cmp v0.4.0
 	github.com/googleapis/gax-go v1.0.3
 	github.com/googleapis/gax-go/v2 v2.0.5
 	github.com/imdario/mergo v0.3.9
 	github.com/mitchellh/copystructure v1.0.0
 	github.com/pkg/errors v0.9.1
+	golang.org/x/crypto v0.0.0-20201002170205-7f63de1d35b0 // indirect
```

Comment:
This seems unexpected 🤔

Reply:
It does, but I couldn't get it to go away. Made sure to kill gopls tidy modules, etc.

Reply:
Hmm did you try wiping out the change, closing everything, then running `go mod tidy`? If not, maybe it is needed. I'll go ahead and approve in case.

Reply:
Nevermind, tried again today and managed to get rid of it.